angular-ngrx-scss/src/app/home/user-gists/user-gists.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <h3>Gists</h3>
+  <ng-container *ngIf="userGists$ | async as gists">
+    <ng-container *ngIf="gists.length > 0">
+      <div class="list-container">
```

Comment:
I agree with you, but wondering if it might be worth making that adjustment in a separate PR, since this one is already so over scope and I know we have the ticket for adding this view still open? #186 

The gists panel also doesn't have the mobile stylings here (to match the NextJS version of the site), so both those updates could be handled in that ticket instead.

Reply:
So long as we leave a note of this somewhere that can be found, I am fine with this being done later (personally).